todoapp/users/models.py

```
@@ -1,30 +1,50 @@
-from django.contrib.auth.models import PermissionsMixin
 from django.db import models
+from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
+from django.contrib.auth.models import PermissionsMixin
 
 
 class UserManager(BaseUserManager):
-    """
-    Add manager methods here to create user and super user
-    """
-
+    def create_user(self, email, password=None, **extra_fields):
```

Ruff (0.12.2)

Missing return type annotation for public function `create_user`

(ANN201)